protected/views/dataset/view.php

```diff
@@ -36,6 +36,7 @@
                             <h4 class="left-border-title left-border-title-lg"><?= $mainSection->getHeadline()['title']; ?></h4>
                             <p class="dataset-release-date-text">Dataset type:  <?= $mainSection->getHeadline()['types'];?> <br> Data released on <?= $mainSection->getHeadline()['release_date'] ?></p>
                             <div class="color-background color-background-block dataset-color-background-block">
+                                <meta name="citation_doi" content="<?php echo $mainSection->getReleaseDetails()['full_doi']; ?>">
```

Comment:
This `meta` tag appears somewhere within `<body>` in the HTML page. Since the `citation_doi` information is metadata within a `<meta>` tag, I think it is better located within the [`<head>`](https://www.w3schools.com/tags/tag_head.asp) section.

If we look at examples from other journals, e.g. this [one](https://www.cell.com/ajhg/fulltext/S0002-9297(17)30074-5) they also have the `meta` tag for `citation_doi` located in `<head>`.